packages/deployer/export-env.sh

```diff
@@ -0,0 +1,3 @@
+# !/bin/bash
```

Comment:
You need to run this script to export your AWS credentials before running the npm scripts locally